scss/forms/_form-control.scss

```diff
@@ -191,7 +191,24 @@ textarea {
 .form-control-color {
   width: $form-color-width;
   height: $input-height;
-  padding: $input-padding-y;
+  padding: $form-color-padding; // Boosted mod
+  border: 2px solid var(--#{$prefix}body-color);
+
+  // Boosted mod
+  &:focus {
+    &[data-focus-visible-added] {
+      background-color: $black;
+    }
+  }
+
+  &:not(:disabled):not([readonly]):hover {
+    background-color: var(--#{$prefix}body-color);
+  }
+
+  &:disabled {
+    filter: grayscale(1);
```

Comment:
Missing border-color change here (maybe need to be forced in `.form-control:disabled` but need to study the impact).
This filter could be changed to `filter: brightness(0) invert(1) brightness(0.4);`.